src/hint.js

```diff
@@ -57,8 +59,48 @@ CodeMirror.registerHelper('hint', 'graphql', (editor, options) => {
 
   var cur = editor.getCursor();
   var token = editor.getTokenAt(cur);
-  var typeInfo = getTypeInfo(schema, token.state);
 
+  const results = getHints(cur, token, schema, editor.getValue(), editor);
+  if (results.list && results.list.length > 0) {
+    results.from = CodeMirror.Pos(results.from.line, results.from.column);
+    results.to = CodeMirror.Pos(results.to.line, results.to.column);
+  }
+  CodeMirror.signal(editor, 'hasCompletion', editor, results, token);
+
+  return results;
+});
+
+// Finds all fragment definition ASTs in a source.
+function getFragmentDefinitions(sourceText) {
```

Comment:
Minor: If you move this back to the same place in the file the git diff will be smaller and git blame will be preserved.